traits/observers/_filtered_trait_observer.py

```diff
@@ -63,19 +65,8 @@ def iter_observables(self, object):
         Yields
         ------
         IObservable
-
-        Raises
-        ------
-        ValueError
-            If the given object cannot be handled by this observer.
         """
-        try:
-            name_to_ctrait = object.traits()
-        except AttributeError:
-            raise ValueError(
-                "Unable to obtain trait definitions from {!r}".format(object)
-            )
-
+        name_to_ctrait = object.traits()
```

Comment:
Could even inline this definition at this point, but it's fine as-is.

Reply:
That's true!